crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1403,7 +1408,7 @@ impl<'db> Type<'db> {
             // `Any` is callable, and its return type is also `Any`.
             Type::Any => CallOutcome::callable(Type::Any),
 
-            Type::Todo(_) => CallOutcome::callable(todo_type!()),
+            Type::Todo(_) => CallOutcome::callable(todo_type!("call type")),
```

Comment:
"call type" doesn't really make sense here, "call todo" would make more sense
```suggestion
            Type::Todo(_) => CallOutcome::callable(todo_type!("call todo")),
```

Reply:
We could also just propagate the existing TODO from the left-hand side